config/annotations.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const Joi = require('joi');
+
+// List of data, eventually key-value pairs
+const SCHEMA_ANNOTATIONS = Joi.object().optional();
```

Comment:
I'm not actually sure which would be preferable, just something that struck me as slightly odd.